client/src/shared/SignUpInputHandler.tsx

```diff
@@ -0,0 +1,34 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { phoneRegExp } from "./AddOfferInputHandler";
+import { passwordRegExp } from "./LoginInputHandler";
+
+const signUpSchema = Yup.object({
+  email: Yup.string().email("Invalid email").required("Email is required"),
+  phone: Yup.string().matches(phoneRegExp, "Phone number is not valid"),
+  city: Yup.string(),
+  password: Yup.string()
+    .required("Password is required")
+    .min(4, "Password is too short - should be 4 chars minimum")
```

Comment:
I think something here is wrong, firstly password must be minimum 4 characters and then 8 characters? :D